x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -214,7 +235,13 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
   }
 
   public start(core: CoreStart, plugins: StartPlugins) {
-    KibanaServices.init({ ...core, ...plugins, kibanaVersion: this.kibanaVersion });
+    KibanaServices.init({
+      ...core,
+      ...plugins,
+      kibanaBranch: this.kibanaBranch,
+      kibanaVersion: this.kibanaVersion,
+      prebuiltRulesPackageVersion: this.prebuiltRulesPackageVersion,
```

Comment:
Nit: It looks a bit disjoined from the rest of the config passed to the FE. I think that we could probably combine `ExperimentalFeaturesService` with the rest of the config and simply pass the whole parsed config to `KibanaServices.init`. No more manual passing of new config settings would be needed when we add some new settings again. Just a thought for future modifications, not suggesting refactoring it within this PR!

Reply:
++, this would be a nice QoL improvement here for managing these configs. Would be curious to revisit how other plugins are doing it as well these days (between client/server). cc @elastic/security-solution-platform